packages/core/src/resolveTrustChains/policies/errors/PolicyMergeError.ts

```diff
@@ -0,0 +1,17 @@
+import type { MetadataPolicyOperator } from '../../../metadata'
+
+type Details = {
```

Comment:
I think if we expose the details (it's marked public) the type should be exported (and thus probably be renamed to something like `PolicyMergeErrorDetails`